### PR TITLE
 Allowing training without test set

### LINKED ISSUE #104: Allowing training without test set
The following patch allows training without having a test set, i.e. for cases where the test set is seperate:
```
diff --git a/alignn/data.py b/alignn/data.py
index 175b915..e70bebc 100644
--- a/alignn/data.py
+++ b/alignn/data.py
@@ -171,8 +171,9 @@ def get_id_train_val_test(
     # full train/val test split
     # ids = ids[::-1]
     id_train = ids[:n_train]
-    id_val = ids[-(n_val + n_test) : -n_test]  # noqa:E203
-    id_test = ids[-n_test:]
+    id_val = ids[-(n_val + n_test) : -n_test]  if n_test > 0 else ids[-(n_val + n_test) :] # noqa:E203
+    id_test = ids[n_test:] if n_test > 0 else []
+
     return id_train, id_val, id_test
 
 
@@ -508,7 +509,7 @@ def get_train_val_loaders(
             classification=classification_threshold is not None,
             output_dir=output_dir,
             tmp_name="test_data",
-        )
+        ) if len(dataset_test) > 0 else None
 
         collate_fn = train_data.collate
         # print("line_graph,line_dih_graph", line_graph, line_dih_graph)
@@ -528,7 +529,7 @@ def get_train_val_loaders(
 
         val_loader = DataLoader(
             val_data,
-            batch_size=batch_size,
+            batch_size=1,
             shuffle=False,
             collate_fn=collate_fn,
             drop_last=True,

```

### PR DESCRIPTION
**Pull request by pbenner**
Fixes #104

### CODE REVIEW

**Comment by knc6** [comment]
Thanks @pbenner for all of the contribution you made recently.
